wpk/common/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM debian:9
 
 RUN apt-get update && \
-    apt-get -y install python git curl jq python3 python3-pip libffi-dev && \
+    apt-get -y install --allow-change-held-packages python git curl jq python3 python3-pip libffi-dev && \
```

Comment:
Why use this instead of `--force-yes`?

Reply:
The `--force-yes` option was splitted since `apt` v1.1 in different `--allow` options. [Here](https://manpages.ubuntu.com/manpages/impish/man8/apt-get.8.html) is the related documentation. 

The proposed option is `--allow-change-held-packages` in Dockerfiles that use the apt version mentioned before, which fixes the issue.

The `--force-yes` can be specified, but a warning will be displayed. Besides, if in the future this option is removed from the `apt` command, these Dockerfiles will have errors again. Replacing it with the `--allow` option avoids the warning, and unless the apt command suffers an downgrade, this option won't generate errors.